python/pyarrow/tests/parquet/test_encryption.py

```diff
@@ -109,7 +109,7 @@ def write_encrypted_parquet(path, table, encryption_config,
                             kms_connection_config, crypto_factory):
     file_encryption_properties = crypto_factory.file_encryption_properties(
         kms_connection_config, encryption_config)
-    assert(file_encryption_properties is not None)
+    assert (file_encryption_properties is not None)
```

Comment:
No need for parentheses around assert statements. Can you make the change in other instances below as well?
```suggestion
    assert file_encryption_properties is not None
```